src/components/ui/InteractWithBurner.tsx

```diff
@@ -0,0 +1,184 @@
+import { AccentText, StatusText } from 'components/ui/Text'
+import { displayFrom, displayTo } from 'helpers/visibilityClassnames'
+import { useState } from 'preact/hooks'
+import Arrow from 'icons/Arrow'
+import Button from 'components/ui/Button'
+import ButtonProps from 'models/ButtonProps'
+import InfoSeal from 'icons/InfoSeal'
+import SmallInfoSeal from 'icons/SmallInfoSeal'
+import classnames, {
+  alignItems,
+  backgroundColor,
+  borderRadius,
+  display,
+  dropShadow,
+  flexDirection,
+  gap,
+  inset,
+  justifyContent,
+  margin,
+  maxWidth,
+  padding,
+  position,
+  textColor,
+  width,
+  zIndex,
+} from 'classnames/tailwind'
+
+const backgroundWrapper = classnames(
+  backgroundColor('bg-primary-dimmed'),
+  padding('p-6'),
+  display('flex'),
+  borderRadius('rounded-lg'),
+  maxWidth('max-w-full', 'lg:max-w-body'),
+  width('w-full', 'md:w-body'),
+  dropShadow('drop-shadow-info-card')
+)
+
+const buttonsWrapper = classnames(
+  display('flex'),
+  alignItems('items-center'),
+  margin('mt-4'),
+  gap('gap-x-4')
+)
+
+const positionWrapper = classnames(
+  display('flex'),
+  flexDirection('flex-col'),
+  width('w-full')
+)
+
+const headerTextWrapper = (show: boolean) =>
+  classnames(
+    display('flex'),
+    gap('gap-x-1'),
+    justifyContent('justify-between'),
+    alignItems(
+      show ? alignItems('items-center', 'md:items-start') : 'items-start'
```

Comment:
Use object instead of calling the function twice:
```TS
alignItems({center: show, md:start: show, start: !show})
```

Reply:
Or
```TS
alignItems(show ? { center: true, md:start: true } : start
```

Reply:
ty, fixed